### ISSUE TITLE
Could you please share some hyperparameters for Resnet-50 in several datasets?

### ISSUE THREAD

**Issue by wzziqian** [KaiyangZhou/deep-person-reid#74]
I am really grateful towards your project and I am just starting Person-Reid Project as a fresh man. 
However, except some setting you shared or default settings, the Models(Resnet50) I tranined have a great distances worse than benchmark.

Could please share the hyperparameters of Resnet50 with xent loss  in  Market1501 | CUHK03 | DukeMTMC-reID | MSMT17 datasets ? 

I am really appreciated if you can reply to me.

**Comment by wzziqian** [comment]
> [#41 (comment)](https://github.com/KaiyangZhou/deep-person-reid/issues/41#issuecomment-408666281)

Thanks for your reply and I have searched this comment before and it this the same setting towards CUHK03 and Duke as well as MSMT17? 

As I have tried all these datasets but nearly 5-15% worse than your shared benchmark， especially MSMT17.

Could you please share the rest of the  hyperparameters  in CUHK03 and DukeMTMCreid and MSMt17?

Thanks a lot

**Comment by wzziqian** [comment]
> what commands did you use to get your results?

I use the default command:
-a resnet50 --optim adam --lr 0.0003 --max-epoch 60 --stepsize 20 40 --train-batch 32 --test-batch 100 --save-dir log/resnet50-xent-market1501 --gpu-devices 0

Especially worse on  MSMT17.
Could you please share the command you used for MSMT17 and CUHK03 and DUKE?  I will cite this project in my work.

**Comment by KaiyangZhou** [comment]
keep it the same and add '--label-smooth'

**Comment by KaiyangZhou** [comment]
Update: training scripts come along with the model weights in the [model zoo](https://github.com/KaiyangZhou/deep-person-reid/blob/master/MODEL_ZOO.md).
